### PR TITLE
Adds New CentCom Clothing Sprites!

### PR DESCRIPTION
**Pull request by Imaginos16**
## About The Pull Request
This PR adds five new clothing options for CentCom characters to look more stylish and aesthetically presentable when
any CentCom related shenanigans happen! It also removes the old CentCom Officer jumpsuits in favor of the new Turtlenecks and outfits to be added.

Now, by hacking the clothesmate, you can buy fake centcom turtlenecks! Have fun pranking security and command by wearing it about! ...Until they notice the very glaring Donk Co. logo. (Thanks Cacogen for the suggestion!)

CentCom Clothes

Of the clothes added, these are:
-CentCom Commander Cap
-CentCom Turtleneck (Real and Replica)
-CentCom Turtleneck Skirt (Real and Replica)
-CentCom Formal Coat
-CentCom Officer's Jumpskirt

## Why It's Good For The Game
The current possible outfits that admins can use for their centcom characters are lackluster and outdated. This PR seeks to rectify
this issue by adding new customization options in the form of thematic clothing, hopefully giving that extra bit of aesthetic appeal to what is a very important job!

## Changelog
:cl:
add: NanoTrasen has now graciously created new outfits for their esteemed members of Central Command! These include new turtlenecks, a fancy coat, a new cap, and a skirt edition of the current Official jumpsuit.
imagedel: As a result of these changes of apparel, however, NanoTrasen came to the decision to remove the old CentCom Officer Jumpsuits in favor of the Turtlenecks.
/:cl:

### CODE REVIEW

**Comment by cacogen** [comment]
ok

**Comment by CRITAWAKETS** [comment]
rebasing in a conflict like this tends to fix it

**Comment by Imaginos16** [comment]
Yeah, sorry about those. I'm trying my best to fix them however!

**Comment by cacogen** [comment]
I wish there were costume variants of these

**Comment by maxymax13** [comment]
cute

**Comment by Imaginos16** [comment]
After a lot of hard work, its ready now. @Twaticus as per your request, the old officer jumpsuits are gonna go bye bye.

**Comment by Imaginos16** [comment]
> I wish there were costume variants of these

That suggestion was so good I'm actually gonna go through the trouble to add these. Thanks!

**Comment by cacogen** [comment]
That's great, thanks

**Comment by Fikou** [comment]
not sure if removing the old jumpsuits is a good idea since they are used by ERT members

**Comment by Imaginos16** [comment]
> not sure if removing the old jumpsuits is a good idea since they are used by ERT members

The change would only affect CentCom inspectors, since if I recall correctly, ERTs use the basic CentCom Official Jumpsuit as is.

**Comment by Imaginos16** [comment]
Alright, now it's ready. Merge?

**Review by Twaticus — APPROVED** [review]

**Comment by Fikou** [comment]
> ERTs use the basic CentCom Official Jumpsuit as is.

Which you removed

**Comment by maxymax13** [comment]
> 
> 
> > ERTs use the basic CentCom Official Jumpsuit as is.
> 
> Which you removed

image

**Comment by Fikou** [comment]
> > > ERTs use the basic CentCom Official Jumpsuit as is.
> > 
> > 
> > Which you removed
> 
> image

they dont use these though 🤔 they use the official ones

**Comment by Imaginos16** [comment]
> > > > ERTs use the basic CentCom Official Jumpsuit as is.
> > > 
> > > 
> > > Which you removed
> > 
> > 
> > image
> 
> they dont use these though these are the commander ones 🤔 they use the official ones

Whoops, guess I was wrong then! I'll test to make sure nothing's wrong, but in theory, the turtlenecks should replace the old officer outfits with no issues.

**Comment by Imaginos16** [comment]
Yep, they replace it without a hitch.
image

**Comment by Imaginos16** [comment]
Alright, I'm ready for merging!

**Comment by Dorsisdwarf** [comment]
I'm confused. This PR removes the jumpsuit and adds a jumpskirt that looks like the removed jumpsuit?

**Comment by Imaginos16** [comment]
> I'm confused. This PR removes the jumpsuit and adds a jumpskirt that looks like the removed jumpsuit?

No no, this one actually removes the officer jumpsuit. The officer jumpsuit, is the one that ERTs used to wear, and what inspectors also wear and are very commonly known for.

**Comment by Dorsisdwarf** [comment]
> 
> 
> > I'm confused. This PR removes the jumpsuit and adds a jumpskirt that looks like the removed jumpsuit?
> 
> No no, this one actually removes the officer jumpsuit. The officer jumpsuit, is the one that ERTs used to wear, and what inspectors also wear and are very commonly known for.

Yes but while I'm confused about why that jumpsuit is being removed, I'm extra confused that you're adding a jumpskirt version of the removed jumpsuit

**Comment by Twaticus** [comment]
its not being removed. lime green centcom uniform is unaffected. lime green centcom skirt is added. tealish green centcom uniform is replaced with a green turtleneck.  green turtleneck skirt is added.

File: code/modules/clothing/head/misc.dm [hunk 1]
```diff
@@ -490,3 +490,12 @@
...
 	inhand_icon_state = "weddingveil"
+
+/obj/item/clothing/head/centom_cap
+	name = "\improper CentCom Commander Cap"
+	icon_state = "centcom_cap"
+	desc = "Worn by the finest of CentCom Commanders. Inside the lining of the cap, lies two faint initials."
+	inhand_icon_state = "that"
+	flags_inv = 0
+	armor = list(MELEE = 30, BULLET = 15, LASER = 30, ENERGY = 40, BOMB = 25, BIO = 0, RAD = 0, FIRE = 50, ACID = 50)
+	strip_delay = 80
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/clothing/head/misc.dm:501]:
```suggestion
	strip_delay = (8 SECONDS)
```

**Comment by SweptWasTaken** [comment]
Looks dope

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-4). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Comment by Twaticus** [comment]
grrr shut up bot

**Comment by ExcessiveUseOfCobblestone** [comment]
sorry buddy ur gbp is now cringe

**Pull request merged**